src/libs/getClickedTargetLocation/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns the Bounding Rectangle for the passed native event's target.
+ *
+ * @param {Object} target
+ * @returns {Object}
+ */
+function getClickedTargetLocation(target) {
```

Comment:
@liyamahendra I thought we'll fix the existing function instead of adding a new one also fix this on the KYCWall page?

Reply:
`getClickedElementLocation` is only used at two places, so it should be safe to refactor this function. We have a conversation going on regarding `KYCWall`, will do the needful accordingly.

Quick question: I'm towards the opinion that`getClickedTargetLocation` is a more appropriate name rather than `getClickedElementLocation`, what do you think? @mananjadhav

Reply:
> `getClickedTargetLocation` is a more appropriate name

I agree. Let's discard the old one.

Reply:
I am OK with where this landed, but I think @mananjadhav's first instinct was correct here. Better to focus on the bug fix first and refactoring later.